Source/GameEngine/Models/Pawn.cs

```diff
@@ -52,26 +46,35 @@ private static void ShowPawnColorMenu()
 
         private static string SetColorOnPawn(string userChoice)
```

Comment:
Why does this method return a value? Shouldn't it just update an internal state of the pawn?
Would i make sense to have an inherited pawn? Like: RedPawn which holds the color and inital posion